meta-mender-core/recipes-mender/mender-connect/mender-connect_2.0.0.bb

```diff
@@ -0,0 +1,30 @@
+require mender-connect.inc
+
+RDEPENDS_${PN} = "glib-2.0 mender-client (>= 2.5)"
```

Comment:
This is not correct, `mender-connect` now requires `mender-client (>= 3.2)` which is already the default in the `.inc` file. So this line must be removed.
```suggestion
```